src/Tests/Core.Tests/CommandTests.cs

```diff
@@ -286,6 +286,17 @@ public async Task GiveOtherItemCommand()
             Count = 10
         }, cfg => cfg.Including(x => x.ItemId).Including(x => x.Count));
     }
+    
+    [Fact]
+    public async Task GiveItemCommand_InvalidPlayer()
+    {
+        await _commandManager.Handle(_connection, "/give missing 1 10");
+
+        (_connection as MockedGameConnection).SentMessages.Should().ContainEquivalentOf(new ChatOutcoming
```

Comment:
Use a direct cast here to avoid warnings :)

Reply:
No. Like this:

```cs
((MockedGameConnection)_connection).SentMessages.Should().ContainEquivalentOf(...)
```

Reply:
Thanks for the explanation! 😄  It's been a while since I last used direct casting as such

Reply:
Its not like the old is obsolete or anything. They serve different purposes :)